utils/MonitoredResource.md

```diff
@@ -0,0 +1,76 @@
+# Monitored Resource
+
+This document is about the monitored resource, what resources are supported and how to automatically
+detect these resources.
+
+## Supported Monitored Resources
+
+* GCP_GCE_INSTANCE
+  * gcp_account: The GCP account number for the instance.
+  * instance_id: The numeric VM instance identifier assigned by GCE.
+  * zone: The GCE zone in which the VM is running.
+* GCP_GKE_CONTAINER
+  * gcp_account: The GCP account number for the instance.
+  * cluster_name: The name for the cluster the container is running in.
+  * namespace_id: The identifier for the cluster namespace the container is running in.
```

Comment:
IIUC namespace_id and container_name should be optional. We cannot get their value from environment variables unless k8s users expose them via the [Downward API](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/).

Refer to k8s document: https://cloud.google.com/kubernetes-engine/docs/tutorials/custom-metrics-autoscaling#exporting_metrics_from_the_application and code sample: https://cloud.google.com/kubernetes-engine/docs/tutorials/custom-metrics-autoscaling#exporting_metrics_from_the_application.

Reply:
Done.